spec/features/user_visits_trucks_index_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+RSpec.describe "User visits trucks index page" do
+  scenario "User sees list of all trucks" do
+    foodTruck = Truck.create(name: 'Yummy Food', description: 'The yummiest food', email: 'yummy@food.com')
```

Comment:
This is the wrong syntax. You need to use snakecase instead of camelcase for Ruby.. Aka `food_truck` instead of `foodTruck`.